### ISSUE TITLE
The First Descendant (2074920)

### ISSUE THREAD

**Issue by nimbold** [ValveSoftware/Proton#7873]
# Compatibility Report
- Name of the game with compatibility issues: The First Descendant
- Steam AppID of the game: 2074920

## System Information
- GPU: GPU0 = Nvidia GTX 1650  ---- GPU1= AMD Vega (integrated)
- Video driver version: Nvidia 555.58.02
- Kernel version: Linux 6.9.7-3-cachyos
- Link to full system information report as [Gist](https://gist.github.com/): (https://gist.github.com/nimbold/e27138959b32d0dd03b15f612bba1186)
- Proton version: Proton Experimental (Also tried with Proton-GE)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-2074920.log](https://github.com/user-attachments/files/16102095/steam-2074920.log)

## Symptoms 
The game always runs on iGPU instead of Dedicated GPU. I tried multiple launch commands, but I didn't have any luck. I tried these:
`__NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia %command%`
and `prime-run %command%`
## Reproduction

1. Open Steam and run the game
2. Try with no launch command or launch commands mentioned above
3. The game runs on iGPU no matter the launch command and therefor the perfromance is ugly

**Comment by nimbold** [comment]
I found the workaround fix thanks to a Nobara linux discord user.
We can fix this by **vulkan icd loader** method.
I just added this in launch command and it works now:
`DXVK_FILTER_DEVICE_NAME="NVIDIA GeForce GTX 1650" VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/nvidia_icd.json %command%`

**Comment by froz3n** [comment]
Just want to report that deleting prefix then starting the game can cause black screen and the game won't start. Seems the game failed to create user config file and can't start.

I crashed and got blackscreen. Nuking prefix and verifying files didn't help. The thing that fixed my black screen problem is this : https://steamcommunity.com/app/2074920/discussions/0/4408543731839110883/

**Comment by kisak-valve** [comment]
The First Descendant: After 1.0.2 patch refuses to launch

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7889.
@maxsupermanhd posted on 2024-07-11T23:03:55:

# Compatibility Report
- Name of the game with compatibility issues: The First Descendant
- Steam AppID of the game: 2074920

## System Information
- GPU: NVIDIA GeForce RTX 4060
- Video driver version: nvidia-open 555.58.02
- Kernel version: Linux 6.9.8-arch1-1
- Link to full system information report: [Gist](https://gist.github.com/maxsupermanhd/5bb0cde87543f1afb0bed9212cfd5c1a)
- Proton version: 7, 8, 9, Experimental (others not tested)

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-2074920.log](https://github.com/user-attachments/files/16184439/steam-2074920.log)

## Symptoms

It does not launch.

Right after opening the window (0 frames drawn) it either closes immediately or hangs in infinite loop consuming 2 cores of the CPU.

## Reproduction

Start the game

**Comment by maxsupermanhd** [comment]
Log with the crash: [steam-2074920.log](https://github.com/user-attachments/files/16184841/steam-2074920.log) (log with infinite loop in message above)

**Comment by maxsupermanhd** [comment]
In console_log.txt in some cases a line
```
naries/Win64/M1-Win64-Shipping.exe: ../../src/xcb_io.c:626: _XAllocID: Assertion `ret != inval_id' failed.
```
appears upon crash, no idea how to fix/approach it really...

**Comment by alligatorshoes** [comment]
Working fine for me here on GE-Proton 9.9.

**Comment by maxsupermanhd** [comment]
Just installed GE-Proton 9.10, same results, I think it is something to do with the game, not Proton

**Comment by froz3n** [comment]
Shot in the dark, but @maxsupermanhd have you tried creating config file like in the link on my post ?

**Comment by maxsupermanhd** [comment]
> Shot in the dark, but @maxsupermanhd have you tried creating config file like in the link on my post ?

Yes, it did nothing, let alone it crates one by itself and still goes into infinite loop or crashes in xcb_io.c.

Also tried running in Bottles but not much success, identical results...

**Comment by maxsupermanhd** [comment]
Is there a way to intercept crash report data or gather some logs that would explain what exactly game is trying to do and where it is failing?

**Comment by froz3n** [comment]
@maxsupermanhd Does PROTON_LOG=1 do nothing ?

Also seems on Linux there's a disconnection problem (code NE:1000). Someone [here](https://www.protondb.com/app/2074920#jMU9S7i-GS) said it's an EAC error but I don't know how to test the fix on fedora.

**Comment by maxsupermanhd** [comment]
> Does PROTON_LOG=1 do nothing ?

Well not nothing, it generates a log with nothing interesting inside. The actual crash message is not being logged in it, it is instead found in console_log.txt file, see my comment above. Just re-tested it and it is absolutely identical despite having couple of updates. I wonder if my system has too new libraries/drives... Might try to run it on my ubuntu laptop for giggles since I believe there are people for who it "just works"

**Comment by maxsupermanhd** [comment]
Indeed, after steam transferred the game via my fast local network to my very slow laptop that can barely count as a thin client for streaming games, it started up perfectly, although with expected rock bottom unplayable performance. I had doubts it will even have enough vram.

So are there any other people on current Arch here with the game running?

**Comment by maxsupermanhd** [comment]
Ok, even more troubleshooting from my side, I side-rolled Fedora Workstation 40 and it launched there just fine. Current Arch borked? Tomorrow will maybe try rolling back whole system to pre-patch date and see what happens...

**Comment by maxsupermanhd** [comment]
Ok so stars lined up real cool, I imagine TFD decided to grab my AMD igpu for upscaling or something (I have one in my ryzen cpu) but it is disabled on bios level so it was just falling over, after adding `DXVK_FILTER_DEVICE_NAME="NVIDIA GeForce RTX 4060"` I presume it decided not to do that anymore and launched just fine... Case closed.

P.S. when tested on Fedora performance was literal seconds per frame so I bet it somehow succeeded in what it tried to do and utilized cpu/igpu to upscale somehow

**Comment by tomByrer** [comment]
I had First Descendant working for a few hours, worked good aside from a few disconnects.  But then it stopped working, even after a simple reinstall.

**Comment by patlefort** [comment]
I am able to run it on my laptop (Dell G5 15 SE), but it won't on my desktop (threadripper 1950x and radeon 6900xt). Both are running Arch Linux. The game will show the initial load screen and then just exit. I tried proton experimental and GE proton. Log shows no obvious problem or crash: [steam-2074920.log](https://github.com/user-attachments/files/17026727/steam-2074920.log)

**Comment by kisak-valve** [comment]
The First Descendant (2074920) complains about no anti-cheat and exits

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9633.
@KenSharp posted on 2026-03-31T18:29:26:

# Compatibility Report
- Name of the game with compatibility issues: The First Descendant
- Steam AppID of the game: 2074920

## System Information
- GPU: Nvidia Quadro K5000
- Video driver version: Nvidia 470.256.02
- Kernel version: Ubuntu 6.18.0
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/KenSharp/bbd42a18faf3803eadeeb9f679896351
- Proton version: Default `9.0 (Beta)`
Also tried 7.0, 10.x and Experimental, no change.

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 

Immediately upon launch the game reports that no anti-cheat is found for this system, and refuses to continue.

[steam-2074920.log.zip](https://github.com/user-attachments/files/26386432/steam-2074920.log.zip)

**Comment by narleen** [comment]
The First Descendant: After the removal of EAC in patch 1.4.3 the game crashes at log-in

### Compatibility Report
* **Name of the game with compatibility issues:** The First Descendant
* **Steam AppID of the game:** 2074920

### System Information
* **GPU:** NVIDIA GeForce RTX 3050
* **Video driver version:** Mesa 26.2.3 (NVK / Zink - Vulkan 1.4)
* **Kernel version:** Linux 7.2.4-202.nobara.fc44.x86_64
* **Link to full system information report:** https://gist.github.com/narleen/dd015c396f2caa4e6ee14225c2071837
* **Proton version:** GE-Proton / Proton Experimental Bleeding-Edge

- [x] I confirm that I haven't found an existing compatibility report for this game.
- [x] I confirm that I have checked whether there are updates for my system available.

### Symptoms
Following the removal of Easy Anti-Cheat (EAC) in patch 1.4.3, the game boots to the title screen but crashes immediately during the login sequence. 

Local crash logs show a `{"message":"Missing Authentication Token"}` response.

### Reproduction
1. Launch the game via Steam on Linux (Proton).
2. Press enter to proceed past the title screen, then log in.
3. The game either crashes on the loading screen, or you briefly enter the Hub before the game crashes.

Note that fresh installs still attempt to run the EAC install script but bypassing this with a launch option doesn't solve the issue.

[steam-2074920.log](https://github.com/user-attachments/files/32412776/steam-2074920.log)
